ecosystem/sep-0038.md

```diff
@@ -44,7 +91,7 @@ All endpoints respond with content type:
 
 ### GET Info
 
-This endpoint describes the supported Stellar assets and off-chain currencies available for trading. Note that the anchor may not support a trading pair between every Stellar asset and off-chain currency listed. Use the [`GET /prices`](#get-prices) endpoint to see which pairs are supported.
+This endpoint describes the supported Stellar assets and off-chain assets available for trading. Note that the anchor may not support a trading pair between every Stellar asset and off-chain asset listed. Use the [`GET /prices`](#get-prices) endpoint to see which pairs are supported.
```

Comment:
❔  Is on-chain/off-chain the appropriate terms to use in the Stellar ecosystem, or on-ledger/off-ledger? I don't know the answer to this myself. @tomerweller ?
```suggestion
This endpoint describes the supported Stellar assets and off-chain assets available for trading. Note that the anchor may not support a trading pair between every Stellar asset and off-ledger asset listed. Use the [`GET /prices`](#get-prices) endpoint to see which pairs are supported.
```

Reply:
I've personally never heard the term "off-ledger" but I hear "on-chain" used by Tomer frequently. Let me know if you feel strongly about using "off-ledger".

Reply:
I don't feel strongly, I was just curious as much for my own vocab as for this SEP. If @tomerweller says on-chain, 👍🏻.